xml/remote_administration.xml

```diff
@@ -82,9 +80,9 @@ nodenames[A-B]          attr[=value],attr[=value],...
    <para>
     The command line utility <command>nodeaddr</command> can be used to query data
```

Comment:
Looks like this command was typo'd here:

```suggestion
    The command line utility <command>nodeattr</command> can be used to query data
```

cnf confirms it's a typo, so i am reasonably sure:
```
> cnf nodeattr
The program 'nodeattr' can be found in the following package:
  * genders [ path: /usr/bin/nodeattr, repository: zypp (repo-oss) ]
[...]

> cnf nodeaddr
nodeaddr: command not found  
```